style.css

```diff
@@ -89,3 +90,303 @@ header {
   list-style: none;
   width: 60%;
 }
+
+.projects {
+  margin: 30% 10px 10px 10px;
+}
+
+.recent-project-name {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+  margin: 15px 0;
+}
+
+.recent-project-img {
+  width: 100%;
+  margin-top: 15%;
+}
+
+.recent-project {
+  margin: 0 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.name {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  color: #fff;
+  text-align: left;
+}
+
+.recent-project-note {
+  font-family: "Inter", serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #091e42;
+  margin-bottom: 20px;
+}
+
+.recent-project-tech {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 20px 0;
+  list-style: none;
+}
+
+.recent-project-tech-used {
+  border: 1px solid #babfc9;
+  padding: 5px 12px;
+}
+
+.see-recent-project {
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 10px 20px;
+  border: none;
+}
+
+.projects-done {
+  margin: 30px 0;
+  background-image: url(./icons/Img-Placeholder.svg);
+  height: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  background-size: cover;
+  width: 100%;
+}
+
+.my-recent-work {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 20px;
+}
+
+.project-note {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #fff;
+}
+
+.project-stack-used {
+  display: flex;
+  list-style: none;
+  width: 80%;
+  justify-content: space-around;
+}
+
+.tech-used {
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  padding: 8px;
+  margin: 10px 0;
+  background-color: #d1dbde63;
+}
+
+.see-project {
+  background-color: #ff6b00;
+  padding: 10px;
+  width: 100%;
+  text-align: center;
+  color: #fff;
+}
```

Comment:
- [ ] Please make sure you are using the same fonts from [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&t=bpXoHDvfJoc09qP5-0) for the `see project` button as according to the requirement we must use the exact same fonts and colors from the design